react-client/src/components/Gallery.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+const Gallery = (props) => {
+    return (
+        <img alt="Massdrop x Fostex TR-X00 Purpleheart Headphones" src="https://helpdesk.massdrop.com/hc/en-us/articles/216723438"/>
+    )
+  }
+  
+  export default Gallery;
```

Comment:
this works as a placeholder for the gallery, still thinking through exactly how we will integrate my component into your content.  we can't use this directly in the browser so I think we will have to assemble our own description html get them working together

Reply:
I think once we are done working on our own components, we'll talk to joe about that?

Reply:
sounds good!